python/decord/video_reader.py

```diff
@@ -36,15 +36,14 @@ class VideoReader(object):
     def __init__(self, uri, ctx=cpu(0), width=-1, height=-1, num_threads=0):
         self._handle = None
         assert isinstance(ctx, DECORDContext)
-        io_type = 0
-        fn = uri
         if isinstance(uri, io.IOBase):
             ba = bytearray(uri.read())
-            self._handle = _CAPI_VideoReaderGetVideoReaderFromBytes(
```

Comment:
Why remove this line? I think the support of read from bytes is a way to solve https://github.com/dmlc/decord/issues/73 . Read from stream byte data

Reply:
it's not removed, your ref link is outdated